README.md

```diff
@@ -115,6 +115,14 @@ If you've globally installed `stylelint-config-standard` using the `-g` flag, th
 }
 ```
 
+Since [stylelint 9.7.0](https://github.com/stylelint/stylelint/blob/9.7.0/CHANGELOG.md#970), you can simply use the globally installed configuration name instead of the absolute path:
+
+```json
+{
+  "extends": "stylelint-config-standard"
+}
+```
+
```

Comment:
Instead of a new section, can you please update the existing doc section on lines 110-116 above